bigbluebutton-html5/imports/api/users/server/modifiers/setConnectionId.js

```diff
@@ -2,9 +2,10 @@ import { check } from 'meteor/check';
 import Users from '/imports/api/users';
 import Logger from '/imports/startup/server/logger';
 
-export default function setConnectionId(meetingId, userId, connectionId) {
+export default function setConnectionId(meetingId, userId, connectionId, authToken) {
```

Comment:
This function should only set the connection id. You need to change the function name or create another one for updating the authToken.